validator/validator-main.protoascii

```diff
@@ -802,50 +802,72 @@ tags: {
 # 4.3.5 The aside element
 tags: {
   tag_name: "ASIDE"
+  # <amp-bind>
```

Comment:
I'm wondering whether we should hold back the actual validator changes until we are very sure we want this format.